docs/tutorials/notebooks/Monge_Gap.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #72.        fig, ax = plt.subplots(figsize=(10, 8))
</pre><p>I'd expose figsize in the arguments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>